Packs/RandomImages_VideosAndAudio/pack_metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Random Images, Videos and Audio",
+    "description": "This packs automation scripts and / or integrations that can pull random photos, videos or audio.\n\nThere is nothing special about this pack. It was designed to lighten up dashboards and incident layouts if so desired.\n\nMore and more scripts and automations will be added over time.\n\nSimply add the automation script to a dynamic-section in an incident layout, or into a script based widget in a dashboard.",
```

Comment:
Everything looks good. But I took out some things that looks like a type-o and deleted 'There iss nothing special about this pack' part. 
And what does 'lighten up dashboards' mean exactly? Sounds a little ambiguous. So I think it will be much better if we change it to 'change the theme of your dashboards'. 

------------------------------
This packs automation scripts and integrations that can pull random photos, videos or audio.\n\nThis pack was designed to change the theme of your dashboards and incident layouts\n\nMore and more scripts and automations will be added over time.\n\nSimply add the automation script to a dynamic-section in an incident layout, or into a script based widget in a dashboard.